docs/API/v3/search-query-service-resource.md

```diff
@@ -0,0 +1,151 @@
+---
+# required metadata 
+
+title: Search, NuGet V3 API | Microsoft Docs
+author:
+- joelverhagen
+- kraigb
+ms.author:
+- joelverhagen
+- kraigb
+manager: skofman
+ms.date: 10/26/2017
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+ms.assetid: 11ca2092-67dc-41a9-a7af-afe610d8febb
+
+# optional metadata
+
+description: The search service allows clients to query for packages by keyword and to filter results on certain package fields.
+keywords: NuGet V3 search API, NuGet V3 discover packages, API to query NuGet packages, API to browse NuGet packages
+ms.reviewer:
+- karann
+- unniravindranathan
+
+---
+
+# Search
+
+It is possible to search for packages available on a package source using the V3 API. The resource used for searching
+is the `SearchQueryService` resource found in the [service index](service-index.md).
+
+## Versioning
+
+There are three resource `@type` values for searching. All three values represent the exact same API surface area. Note
+that non-breaking changes (defined as added JSON properties or query parameters) are periodically added and documented
+here.
+
+The `@type` values are:
+
+ - `SearchQueryService`
+ - `SearchQueryService/3.0.0-beta`
+ - `SearchQueryService/3.0.0-rc`
+
+### API changes
+
+ - August 2017: Added the `verified` property to the search result.
+ - March 2017: Added a `semVerLevel` query parameter which defaults to filtering out SemVer 2.0.0 packages.
```

Comment:
Perhaps make `SemVer 2.0.0 packages` a link to the definition, as it may not be obvious what that actually means: https://github.com/NuGet/Home/wiki/SemVer2-support-for-nuget.org-%28server-side%29#identifying-semver-v200-packages

Reply:
Great point.

Reply:
Fixed.